packages/cinterop/src/jvmCommon/CMakeLists.txt

```diff
@@ -1,54 +1,16 @@
 cmake_minimum_required(VERSION 3.4.1)
+project(RealmKotlin)
 set(CAPI "${CMAKE_SOURCE_DIR}/../../../external/core/")
 set(CAPI_BUILD "${CAPI}/build-android-${ANDROID_ABI}-${CMAKE_BUILD_TYPE}")
 set(SWIG_JNI "${CMAKE_SOURCE_DIR}/../../../jni-swig-stub/src/main/jni")
 
-# Cannot seem to find the libraries with find_library, maybe Android build overrides some internals
-link_directories("${CAPI_BUILD}/src/realm")              # realm
-link_directories("${CAPI_BUILD}/src/realm/object-store") # realm-object-store
-link_directories("${CAPI_BUILD}/src/realm/parser")       # realm-parser
-#link_directories("${CAPI_BUILD}/src/realm/sync")         # realm-sync
-link_directories("${CAPI_BUILD}/src/realm")              # realm-object-store
-link_directories("${CAPI_BUILD}/src/realm/object-store/c_api") # librealm-ffi-dbg.a
-link_directories("${CAPI_BUILD}/openssl/lib/")           # libcrypto.a libssl.a
+# Build Realm Core
+# Set option flags for Core.
+# See https://github.com/realm/realm-core/blob/master/CMakeLists.txt#L174 for the full list.
+set(REALM_ENABLE_SYNC OFF)
+set(REALM_BUILD_LIB_ONLY ON)
+add_subdirectory("realm-core" EXCLUDE_FROM_ALL)
```

Comment:
`add_subdirectory` are required to be a subdir unless you also specify the binary output dir for the CMake project and since Core has quite a lot of output dirs I wasn't sure it was possible. So this was the quickest way around it.

But we probably have to go back for a 2nd when we want to build on Windows.